examples/mobile/UIComponents/components/controllers/dropdownlist/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page">
+		<header>
+			<div class="ui-appbar-left-icons-container">
+				<a href="#" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></a>
+			</div>
+			<div class="ui-appbar-title-container">
+				<span class="ui-appbar-title">Dropdown list</span>
+			</div>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview ui-colored-list">
```

Comment:
ui-colored-list is no longer used. Use the 'ui-content-area' instead of the 'ui-colored-list'.

Reply:
Ok. Thanks, applied to index.html and newly added dropdownlist/align-to-list.html.